torch/csrc/jit/backends/backend_init.cpp

```diff
@@ -0,0 +1,11 @@
+#include <torch/csrc/jit/backends/backend_init.h>
+#include <torch/csrc/jit/backends/test_backend.h>
+
+namespace torch {
+namespace jit {
+
+void initJitBackendBindings(PyObject* module) {
+  initTestBackendBindings(module);
```

Comment:
Should NewBackend include its header here and add initNewBackendBindings(..) ?

Reply:
No, the plan is to move `TestBackend` to the tests directory and have it register its bindings with `torch._C` or a C extension module created just for testing. 

`NewBackend` should register its bindings in its own C extension module.